runtime/testruntime/testruntime.go

```diff
@@ -75,6 +75,21 @@ func NewInstanceWithModel(t TestingT, name string, sql string) (*runtime.Runtime
 	return rt, instanceID
 }
 
+func NewInstanceWithDefaultModel(t TestingT) (*runtime.Runtime, string) {
+	rt, instanceID := NewInstanceWithModel(t, "test", `
+		SELECT 'abc' AS col, 1 AS val, TIMESTAMP '2022-11-01 00:00:00' AS times, DATE '2007-04-01' AS dates
+		UNION ALL 
+		SELECT 'def' AS col, 5 AS val, TIMESTAMP '2022-11-02 00:00:00' AS times, DATE '2009-06-01' AS dates
+		UNION ALL 
+		SELECT 'abc' AS col, 3 AS val, TIMESTAMP '2022-11-03 00:00:00' AS times, DATE '2010-04-11' AS dates
+		UNION ALL 
+		SELECT null AS col, 1 AS val, TIMESTAMP '2022-11-03 00:00:00' AS times, DATE '2010-11-21' AS dates
+		UNION ALL 
+		SELECT 12 AS col, 1 AS val, TIMESTAMP '2022-11-03 00:00:00' AS times, DATE '2011-06-30' AS dates
+	`)
+	return rt, instanceID
+}
```

Comment:
Since this is a test case(s) specific dataset, it feels like this belongs as a helper function in the specific package that uses it. The problem with centralizing too much is that if someone edits this to expand a test case in one place, then they might break many other tests that they didn't want to touch.

(Also, is it currently used anywhere?)

Reply:
Frankly, my point is:
* I'm certain I will reuse it somewhere
* At the point where someone tweaks it and breaks a bunch of unit tests he/she will have a good lesson to check first if it's used anywhere else. After having a lesson he/she can inline it and do a special model for a special unit test.